packages/pytest-simcore/src/pytest_simcore/rabbit_service.py

```diff
@@ -39,37 +39,51 @@ async def wait_till_rabbit_responsive(url: str) -> None:
 
 
 @pytest.fixture(scope="function")
-async def rabbit_config(docker_stack: Dict, testing_environ_vars: Dict) -> RabbitConfig:
+async def rabbit_settings(
+    docker_stack: Dict, testing_environ_vars: Dict  # stack is up
+) -> RabbitSettings:
+    """Returns the settings of a rabbit service that is up and responsive"""
+
     prefix = testing_environ_vars["SWARM_STACK_NAME"]
     assert f"{prefix}_rabbit" in docker_stack["services"]
-    rabbit_config = RabbitConfig(
-        user=testing_environ_vars["RABBIT_USER"],
-        password=testing_environ_vars["RABBIT_PASSWORD"],
-        host="127.0.0.1",
-        port=get_service_published_port("rabbit", testing_environ_vars["RABBIT_PORT"]),
-        channels=json.loads(testing_environ_vars["RABBIT_CHANNELS"]),
+
+    port = get_service_published_port("rabbit", testing_environ_vars["RABBIT_PORT"])
+
+    settings = RabbitSettings(
+        RABBIT_USER=testing_environ_vars["RABBIT_USER"],
+        RABBIT_PASSWORD=testing_environ_vars["RABBIT_PASSWORD"],
+        RABBIT_HOST="127.0.0.1",

```

Comment:
should we not get the host ip here instead of 127.0.0.1? in case this is used from a container as well?

Reply:
good point.